proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -142,6 +142,11 @@ message AuthInfo {
   //
   // Since: cosmos-sdk 0.46
   Tip tip = 3;
+
+  // extension_options are arbitrary options that can be added by chains
+  // when the default options are not sufficient. If any of these are present
+  // and can't be handled, the transaction will be rejected
+  repeated google.protobuf.Any extension_options = 1023;
```

Comment:
I think we wanted to add it to the `Fee` type

Reply:
also, how about renaming `extension_options` to `extension`?

Reply:
I think this is actually the right place and right name

Reply:
So I guess it's fine with this change? 😄

Reply:
Doesn't extension imply that it's optional? Personally I prefer short names. Just a comment ;)